provision-osx/tasks/cocoapods.yml

```diff
@@ -1,13 +1,22 @@
 - include: configure_node_env_vars.yml
 
 -
+  name: Install cocoapods
+  shell: gem install -n /usr/local/bin -v {{ cocoapods_version | default('\"\"') }} cocoapods --no-ri --no-rdoc
+  args:
+    executable: /bin/bash
+  environment:
+    http_proxy: "{{ proxy_url | default('') }}"
+    https_proxy: "{{ proxy_url | default('') }}"
+  become: yes
+
+-
   name: Update Cocoapods
   shell: bash -lc "pod repo update"
   register: pod_repo_update
   changed_when: pod_repo_update.stdout.find('Already up-to-date') == -1
   environment:
     PATH: "{{ buildfarm_env_path.stdout }}"
-    GEM_HOME: "{{ buildfarm_env_gem.stdout }}"
     LANG: "{{ buildfarm_lang_env_var.stdout }}"
```

Comment:
I think this should be just `buildfarm_env_lang` without .stdout OR it should be `buildfarm_env_lang`